src/routes.rs

```diff
@@ -111,3 +117,42 @@ where
         }
     }
 }
+
+async fn log_request_response_body(
```

Comment:
```suggestion
async fn body_logger_middleware(
```
plus move everything to separate file